straxen/contexts.py

```diff
@@ -27,6 +27,11 @@
     n_tpc_pmts=straxen.n_tpc_pmts,
     n_top_pmts=straxen.n_top_pmts,
     gain_model=('to_pe_constant', 'gain_2e6HVmap_cutoff_1310'),
+    tail_veto_threshold=1500.,  # PE
```

Comment:
Thanks Sophia, in general all looks good as far as I'm concerned. Will be an AC call (@feigaodm) when this will be merged.

_just adding the same comment for book keeping_
Could you move the config to `xenonnt_online`?